src/types/FormTypes.ts

```diff
@@ -27,14 +27,35 @@ type TimelineForm = {
   synced: string;
   sessionType: string;
   reportLink: string;
-  reportSchedule: string;
+  repeatSchedule: string;
   startDate: string;
   endDate: string;
   startTime: string;
   endTime: string;
 };
 
-type MyForm = StudentForm | TestForm | TimelineForm;
-type MyFormKey = keyof StudentForm | keyof TestForm | keyof TimelineForm;
+type SessionType = {
+  number_of_fields_in_pop_form: number | null;
+  form_schema_id: number | null;
+  auth_type: null | string;
+  id_generation: boolean | null;
+  activate_signup: boolean | null;
+  redirection: boolean | null;
+  pop_up_form: boolean | null;
+};
+
+type MyForm = StudentForm | TestForm | TimelineForm | SessionType;
```

Comment:
dont call it myform. think of some other name

Reply:
QuizForm or EntityForm